ui/app/styles/core/breadcrumb.scss

```diff
@@ -53,6 +53,9 @@
   }
```

Comment:
A regression. Traced back to overriding issues and copied the exact binary css that is on `breadcrumb a` selectors.